core/src/code/utils.rs

```diff
@@ -36,7 +36,7 @@ use gear_wasm_instrument::{
 };
 
 /// Defines maximal permitted count of memory pages.
-pub const MAX_WASM_PAGE_COUNT: u16 = 512;
+pub const MAX_WASM_PAGE_AMOUNT: u16 = 512;
```

Comment:
What the reason to rename the constant? From my point of view it doesn't make much sense but triggers various edits of the code.

Reply:
Because, count is closer to counter, and amount is closer to quantity. Also soon I am going to add WasmPagesAmount in addition to existed WasmPage